cstar/tests/unit_tests/roms/test_roms_simulation.py

```diff
@@ -1496,64 +1546,63 @@ def test_runtime_code_modifications(self, example_roms_simulation):
         )
 
     def test_runtime_code_modifications_raises_if_no_working_path(
-        self, example_roms_simulation
-    ):
+        self, example_roms_simulation: tuple[ROMSSimulation, Path]
+    ) -> None:
         """Ensures an error is raised if `runtime_code.working_path` is not set.
 
         This test verifies that attempting to access `_runtime_code_modifications`
         without a valid `runtime_code.working_path` results in a `ValueError`.
 
         Mocks & Fixtures
         ----------------
-        - `example_roms_simulation` : Provides a pre-configured `ROMSSimulation`
-          instance.
+        example_roms_simulation : tuple[ROMSSimulation, Path]
+            Provides a pre-configured `ROMSSimulation` instance.
 
         Assertions
         ----------
         - Checks that accessing `_runtime_code_modifications` without setting
           `runtime_code.working_path` raises a `ValueError`.
         """
-
         sim, directory = example_roms_simulation
         with pytest.raises(
             ValueError, match="does not have a 'working_path' attribute."
         ):
-            sim._runtime_code_modifications
+            _ = sim._runtime_code_modifications  # noqa: SLF001
 
     def test_runtime_code_modifications_raises_if_no_template(
-        self, example_roms_simulation
-    ):
+        self, example_roms_simulation: tuple[ROMSSimulation, Path]
+    ) -> None:
         """Ensures an error is raised if no template namelist file is found.
 
         This test verifies that `_runtime_code_modifications` raises a `ValueError`
         when `runtime_code.files` does not contain a template namelist file
         (i.e., a file ending in `_TEMPLATE`).
 
         Mocks & Fixtures
-        ---------------
-        - `example_roms_simulation` : Provides a pre-configured `ROMSSimulation`
-          instance.
+        ----------------
+        example_roms_simulation: tuple[ROMSSimulation, Path]
```

Comment:
revert removal of list formatting

Reply:
pasted a "parameters" version, not a mocks & fixtures version... this should be a list and not be the `name : type` format